src/main/java/com/weavers/duqhun/util/RestApiHitter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.weavers.duqhun.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import net.sf.json.JSONException;
+import net.sf.json.JSONObject;
+
+/**
+ *
+ * @author Android-3
+ */
+public class RestApiHitter {
+
+    //------------------JSON Read By Using Java Dependency (org.json) (net.sf.json-lib)----------------------//http://www.jsonschema2pojo.org/
+    public static void readJsonFromUrl() throws IOException, JSONException {
+        JSONObject JsonData = new JSONObject();
+
+        JsonData.put("MID", "DIY12386817555501617");

```

Comment:
can you put all these constant access tokens for cloudinary, paytm etc into a constant file, so that we can in the future secure them? Instead of having all these tokens everywhere in the code.